common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -122,26 +122,6 @@ object WhiskConfig {
    */
   def readFromEnv(key: String): Option[String] = sys.env.get(asEnvVar(key))
 
-  private def whiskPropertiesFile: File = {
```

Comment:
this file is now only used for tests and tests should use `WhiskProperties` which already have to discover this file.